Gemfile

```diff
@@ -19,6 +19,8 @@ gem 'jbuilder'
 gem 'jquery-rails'
 gem 'jquery-turbolinks'
 gem 'jquery-ui-rails'
+# Only used in a few controllers, could be refactored
+gem 'haml'
```

Comment:
Haml is already included in the form of haml-rails. If you had kept to my nice alphabetic order you would have seen this at once :)